docs/sources/installation/security.md

```diff
@@ -12,29 +12,27 @@ weight = 2
 
 # Security
 
-## Data source proxy and protecting internal services
+If you run non-Grafana web services on your Grafana server or within its local network, then they might be vulnerable to exploitation through the Grafana data source proxy.
```

Comment:
`Data source proxy and protecting internal services` and `Viewer query permissions` are explaining two different things, though to some extent related but `Viewer query permissions` applies to both data source proxy requests and backend data source queries. 

Data source proxy calls: https://grafana.com/docs/grafana/latest/http_api/data_source/#data-source-proxy-calls
Data source backend query seems to be lacking from HTTP API, but should be `/api/tsdb/query`.

With your changes all sections is in H2 with intro text only referring to "exploitation through the Grafana data source proxy" which seems incorrect to me. I would consider having them in two different H2 headers or re-write the intro-text. 

Thoughts?

Reply:
I found that endpoint mentioned here: https://grafana.com/docs/grafana/v6.2/plugins/developing/backend-plugins-guide/#request-format

If something needs to be added to the API section, please put in a PR, even if it is sparse.

I am unclear on what relating to the tsdb query API that you cite should be added here.

I rewrote the introduction, please review and let me know if you have additional suggestions.